packages/plugin-mongo/src/database.ts

```diff
@@ -7,8 +8,12 @@ export interface Config {
   protocol?: string
   host?: string
   port?: number
-  name?: string
+  name?: string // database name
   prefix?: string
+  authDatabase?: string // default auth database
+  connectionOptions?: URLSearchParams | string | { [key: string]: string | (string)[] } | Iterable<[string, string]> | Array<[string, string]>
```

Comment:
@types/node is using `URLSearchParams | string | NodeJS.Dict<string | ReadonlyArray<string>> | Iterable<[string, string]> | ReadonlyArray<[string, string]>`